recipes/mapd-core-cpu/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "mapd-core-cpu" %}
+{% set version = "4.5.0" %}
+{% set sha256 = "d2a0f7ffd4366a38aed2a5a81e69afde5293aa03fdcbf5142ff2caed6cccce5b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mapd/mapd-core/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or py2k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - arrow-cpp>=0.12
+    - bisonpp
+    - blosc
+    - boost-cpp=1.68
+    - boost=1.68
```

Comment:
You shouldn't use exact pins like this, if you can avoid it.

Reply:
thanks @scopatz ! I am not sure if it is working with boos 1.69 that is why I am pinning 1.68 (I changed `=` to ` `  `boost-cpp 1.68`)

Reply:
You should try, though.  You should only use pinnings when the package **requires** it or you know there are breakages.  Otherwise, it is important to let conda-forge-pinning be used, so that the package is in sync with the rest of conda-forge.

Reply:
IIRC, boost-cpp has to be exactly 1.68. Otherwise the mapd-core does not build (it uses a feature that was removed from boost 1.69).
To remove the pinning, the mapd-core should be fixed, but that is not the current issue.

Reply:
thanks @pearu for the explanation!